deploy/infra-cluster-service-account.yaml

```diff
@@ -13,7 +13,7 @@ rules:
   verbs: ["get", "create", "delete"]
 - apiGroups: ["kubevirt.io"]
   resources: ["virtualmachineinstances"]
-  verbs: ["list"]
+  verbs: ["list", "get"]
```

Comment:
when this merges, can you update the hypershift equivalent of this file here, https://github.com/openshift/hypershift/blob/9e49828a874ecff569dcef5d89e046abb48e4195/control-plane-operator/controllers/hostedcontrolplane/csi/kubevirt/files/infra_role.yaml#L12